erpnext/stock/doctype/purchase_receipt/purchase_receipt.js

```diff
@@ -57,6 +57,12 @@ frappe.ui.form.on("Purchase Receipt", {
 				}
 			}
 		});
+		
+		frm.set_query("transporter", function() {
```

Comment:
@rehanansari26 There is no field named `transporter` in PR. There is `transporter_name` which is a Data field, so filters can't applied to it using set_query.

Reply:
then it is created after installation? what is mean by **Is System Generated** ? It is belong to which app ?
![image](https://github.com/frappe/erpnext/assets/110723484/84886fe8-ac25-4108-89f7-bf7ac359b2b5)

Reply:
It might be created by the India Compliance App, need to check.